_data/defis.yml

```
@@ -1209,11 +1209,26 @@
   duration_months: 12
   eigs:
     - "Marie Ramon-Daré"
+  mentors:
+    - "Olivier Garry"
+    - "Thomas Lesueur"
+
+  "Sequoia IGN":
```

```suggestion
"Sequoia IGN":
```
Pas d'espaces ici, ça doit être la source de tes autres erreurs